client/me/purchases/remove-purchase/index.jsx

```diff
@@ -343,7 +345,12 @@ const RemovePurchase = React.createClass( {
 					className="remove-purchase__dialog"
 					isVisible={ this.state.isDialogVisible }
 					onClose={ this.closeDialog }>
-					<FormSectionHeading>{ this.translate( 'Remove %(productName)s', { args: { productName } } ) }</FormSectionHeading>
+					<FormSectionHeading>
+						{ translate( 'Your thoughts are needed.' ) }
+					</FormSectionHeading>
+					<p>
+						{ translate( 'Before you go, please answer a few quick questions to help us improve WordPress.com.' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **0** time:
`translate( 'Before you go, please answer a few quick questions to help us improve WordPress.com.' )` *ES Score: 14*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).